source_code/main_mcu/src/EMU/lis2hh12.c

```diff
@@ -3,18 +3,21 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <time.h>
+#include "emulator.h"
 
 static int urandom = -1;
 
 BOOL lis2hh12_check_data_received_flag_and_arm_other_transfer(accelerometer_descriptor_t* descriptor_pt, BOOL arm_other_transfer)
 {
+    int16_t base_x = emu_get_lefthanded() ? 16384 : -16384;
+    int16_t base_y = 0;
+    int16_t base_z = 0;
     (void)arm_other_transfer;
     for (uint16_t i = 0; i < MEMBER_ARRAY_SIZE(acc_single_fifo_read_t, acc_data_array); i++)
     {
-        /* Only fill the lower bytes as the rng uses them */
```

Comment:
deleting my comment? really? xD

Reply:
Well, after my changes the comment is no longer true :)